### PR TITLE
Use the correct version of Visual Studio for changing the compiler flags

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build

**Review by pcanal — APPROVED** [review]

File: hist/histv7/test/CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
-if(MSVC AND MSVC_VERSION GREATER_EQUAL 1924)
+if(MSVC AND MSVC_VERSION GREATER_EQUAL 1923)
   # FIXME: using /O2 compiler flag prevent the followin error when building in debug mode:
```
Comment (pcanal) [inline hist/histv7/test/CMakeLists.txt:8]:
```suggestion
  # FIXME: using /O2 compiler flag prevent the following error when building in debug mode:
```

File: hist/histv7/test/CMakeLists.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
   # axis.obj : fatal error LNK1179: invalid or corrupt file: duplicate COMDAT
   # '??$?8DU?$char_traits@D@std@@@__ROOT@experimental@std@@YA_NV?$basic_string_view@DU?$char_traits@D@std@@@012@0@Z'
```
Comment (pcanal) [inline hist/histv7/test/CMakeLists.txt:10]: Is there a corresponding MSVC bug report?

Comment (bellenot) [reply hist/histv7/test/CMakeLists.txt:10]: Nope, not sure if it's coming from MSVC, and didn't try to come with a simple reproducer

Comment (pcanal) [reply hist/histv7/test/CMakeLists.txt:10]: fair enough ...

**Pull request merged**